codalab/common.py

```diff
@@ -286,7 +286,7 @@ def _get_azure_sas_url(self, path, **kwargs):
             account_name=AZURE_BLOB_ACCOUNT_NAME,
             container_name=AZURE_BLOB_CONTAINER_NAME,
             account_key=AZURE_BLOB_ACCOUNT_KEY,
-            expiry=datetime.datetime.now() + datetime.timedelta(hours=1),
```

Comment:
Should we refactor this into a constant? Also, is 10 hours enough or might we need more for even larger files?